includes/admin/meta-boxes/class-wc-meta-box-coupon-data.php

```diff
@@ -341,51 +343,54 @@ public static function output( $post ) {
 			</div>
 			<?php do_action( 'woocommerce_coupon_data_panels', $coupon->get_id(), $coupon ); ?>
 			<div class="clear"></div>
-		</div>
```

Comment:
It isn't correct this change.
We need to close `<div id="coupon_options" class="panel-wrap coupon_data">` in here. But this is not what is happening anymore.